### PR TITLE
Add gui

### PR DESCRIPTION
**Pull request by jozw**

### CODE REVIEW

File: lib/rf_footer/deployment_info.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module RfFooter
+  class DeploymentInfo
+    class << self
+      def revision
+        Git::REVISION || "unknown SHA"
+      end
+
+      def environment
+        ENV.fetch("RF_ENV", "undefined environment")
```
Comment (jozw) [inline lib/rf_footer/deployment_info.rb]: @toddrun what do you think about?
```ruby
ENV["RF_ENV"] || ENV["RORY_ENV"] || ENV["RAILS_ENV"] || "undefined environment"
```

Comment (toddrun) [inline lib/rf_footer/deployment_info.rb]: Actually, I think you'll get better results using the full set of options. It's a bit verbose, but it's more complete.

File: spec/rf_footer/deployment_info_spec.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  end
+
+  describe ".environment" do
+    context "environment defined" do
+      before { ENV["RF_ENV"] = "tropical savanna" }
+
+      it "returns environment" do
+        expect(described_class.environment).to eq("tropical savanna")
+      end
+    end
```
Comment (toddrun) [inline spec/rf_footer/deployment_info_spec.rb]: This is where adding the other ENV environment variables is going to hurt . . . it's a lot more test code to verify the whole thing.

So, if RF_ENV is going to be set in prod, staging, beta and sandbox, you're probably good to just go with that.

**Comment by toddrun** [comment]
+1

**Pull request merged**